docs/demo.md

```diff
@@ -33,6 +34,7 @@ alias armadactl='go run cmd/armadactl/main.go --armadaUrl armada.demo.armadaproj
 Create queues, submit some jobs, and monitor progress:
 
 ### Queue Creation
+Change the name of the queue to something unique instead of `queue-a` and `queue-b`. You would need these names in next steps for Job Submission.
```

Comment:
I think you can remove this and just say use a unique name for queue and then use something like QUEUE_NAME as a placeholder.  I think this is common practice rather than explictly assume someone to use queue-a or queue-b.